sweyer_plugin/lib/sweyer_plugin_method_channel.dart

```diff
@@ -138,8 +168,13 @@ class MethodChannelSweyerPlugin extends SweyerPluginPlatform {
   }
 
   @override
-  Future<void> removePlaylists(List<int> playlistIds) async =>
-      methodChannel.invokeMethod<void>('removePlaylists', {'ids': playlistIds});
+  Future<void> removePlaylists(List<int> playlistIds) async {
+    try {
+      return methodChannel.invokeMethod<void>('removePlaylists', {'ids': playlistIds});
```

Comment:
this will be not catched without `await`

Reply:
Good catch, thanks